FrontEnd/main/src/components/cookieacception/CookieMod.js

```diff
@@ -0,0 +1,54 @@
+import { Cookies } from 'react-cookie';
+import { useCookies } from 'react-cookie';
+import styles from './CookieMod.module.css';
+import cookiesimage from './cookieimg/homecookies.png';
+
+const CookieMod = ({ active, setActive }) => {
+  const [cookies, setCookie, removeCookie] = useCookies();
+  console.log(cookies.first);
+  const accessCookie = () => {
+    let d = new Date();
+    d.setTime(d.getTime() + 30 * 24 * 60 * 60 * 1000);
```

Comment:
It would be good to move `30 * 24 * 60 * 60 * 1000` into a constant with a meaningful name, instead of using magic numbers.

Reply:
It really looks better now. Thank you. Done